lib/bundle/locker.rb

```diff
@@ -3,6 +3,7 @@
 require "tap"
 require "os"
 require "development_tools"
+require "env_config"
```

Comment:
Gonna have to stub this out

Reply:
Don't need it in the end since I'm using the full `Homebrew::EnvConfig` directly.

Reply:
Nope, I was wrong, still do, oops. I will fix the tests.